libs/SalesforceSwiftSDK/SalesforceSwiftSDK/Extensions/SFRestAPIExtensions.swift

```diff
@@ -26,13 +26,56 @@ import Foundation
 import PromiseKit
 import SalesforceSDKCore
 
+public struct SFRestResponse {
+    
+    private (set) var data : Data?
+    private (set) var urlResponse : URLResponse?
+    
+    init(data: Data?,response: URLResponse?) {
+        self.data = data
+        self.urlResponse = response
+    }
+    
+    func asJsonDictionary() -> [String: Any] {
```

Comment:
How does one get the raw data (eg for a call returning binary)

Reply:
@wmathurin  Should we add an accessor?  response.data should do it.

Reply:
That would be nice, it would mirror the asBytes() we have on the Android side.